tracer/test/snapshots/WcfTests.__binding=BasicHttpBinding_enableCallTarget=False_enableNewWcfInstrumentation=False.verified.txt

```diff
@@ -0,0 +1,218 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: wcf.request,
+    Resource: WcfSample/ICalculator/ServerSyncAdd,
+    Service: Samples.Wcf,
+    Type: web,
+    ParentId: Id_3,
```

Comment:
Just to clarify, the missing parent spans here are because you filtered them out with?
`.Where(s => !s.Resource.Contains("schemas.xmlsoap.org") && !s.Resource.Contains("www.w3.org"));`
Do we _have_ to filter them out? What's variable about them?

Reply:
The spans that are filtered out with the resource name are WCF spans that I really don't understand. It seems like it's an unpredictable, periodic schema check that occurs outside of any one WCF request. So even if we kept them, they wouldn't be in the same trace as the individual requests we're trying to track.